frontend/src/metabase/selectors/embed.ts

```diff
@@ -12,3 +12,7 @@ export const getEmbedOptions = (state: State): EmbedOptions => {
 export const getIsEmbeddingSdk = (state: State): boolean => {
   return !!state.embed.isEmbeddingSdk;
 };
+
+export const getIsNightMode = (state: State): boolean => {
+  return getEmbedOptions(state)?.theme === "night";
+};
```

Comment:
The problem is that `DashboardControls.jsx` stores the theme globally (with the message confirming what I thought `It should probably be in Redux?` in the comment 😅), but we don't expose it anywhere else...

We currently use a CSS selector in Emotion to do this, which you can't really do outside Emotion (unless you do `document.querySelector` which is not nice). The best solution I can think of is relying on embed options - just expose it.

This will likely break if there is environments outside of embed that uses night mode, but I can't find this setting or button anywhere in the dashboard. A perfect solution would be to lift the `isNightMode` state in `DashboardControls` out to the store to support non-embed use case, but if only embed uses it then this will do.

```ts
  .${DashboardS.Dashboard}.${DashboardS.DashboardNight}.${DashboardS.DashboardFullscreen}
    .${DashboardS.fullscreenNightText}
    &,
  .variation-container-tooltip & {
    color: ${lighten("text-medium", 0.15)};
  }
```

Reply:
Update: I just learned that there is a "nighttime mode" button that is only shown in fullscreen 🤦🏻 

Relying on embed options alone won't work then. The only way to not rely on the class name detection is to lift the isNightMode state from `DashboardControls` to the store 💥

Reply:
Lifting `isNightMode` sounds like a right direction, then passing it as a prop and assigning a class on the root when night mode is enabled. Wdyt?